Index: rdkit/Code/GraphMol/FileParsers/MaeMolSupplier.cpp
===================================================================
--- rdkit.orig/Code/GraphMol/FileParsers/MaeMolSupplier.cpp
+++ rdkit/Code/GraphMol/FileParsers/MaeMolSupplier.cpp
@@ -16,7 +16,7 @@
 #include <GraphMol/MolOps.h>
 #include <GraphMol/RWMol.h>
 #include <GraphMol/FileParsers/MolSupplier.h>
-#include <CoordGen/maeparser/Reader.hpp>
+#include <maeparser/Reader.hpp>
 
 
 namespace RDKit {
Index: rdkit/Code/GraphMol/Wrap/MaeMolSupplier.cpp
===================================================================
--- rdkit.orig/Code/GraphMol/Wrap/MaeMolSupplier.cpp
+++ rdkit/Code/GraphMol/Wrap/MaeMolSupplier.cpp
@@ -20,7 +20,7 @@
 #include <GraphMol/RDKitBase.h>
 #include <RDBoost/python_streambuf.h>
 #include <RDBoost/iterator_next.h>
-#include <CoordGen/maeparser/Reader.hpp>
+#include <maeparser/Reader.hpp>
 
 #include "MolSupplier.h"
 
