--- a/External/CMakeLists.txt
+++ b/External/CMakeLists.txt
@@ -1,4 +1,3 @@
 add_subdirectory(INCHI-API)
 add_subdirectory(AvalonTools)
 add_subdirectory(FreeSASA)
-add_subdirectory(CoordGen)
--- a/External/CoordGen/CMakeLists.txt
+++ b/External/CoordGen/CMakeLists.txt
@@ -1,4 +1,5 @@
-add_custom_target(coordgen_support ALL)
+include_directories(/usr/include/maeparser)
+include_directories(/usr/include/coordgenlibs)
 
 if(RDK_BUILD_COORDGEN_SUPPORT)
     add_definitions(-DIN_MAEPARSER)
@@ -7,56 +8,10 @@
     if(NOT DEFINED MAEPARSER_DIR)
       set(MAEPARSER_DIR "${CMAKE_CURRENT_SOURCE_DIR}/maeparser")
     endif()
-    if(NOT EXISTS "${MAEPARSER_DIR}/MaeParser.hpp")
-        set(RELEASE_NO "1.0.1")
-        downloadAndCheckMD5("https://github.com/schrodinger/maeparser/archive/v${RELEASE_NO}.tar.gz"
-              "${CMAKE_CURRENT_SOURCE_DIR}/master.tar.gz"
-              "1292494df756e95fd1cce722286f28fe")
-        execute_process(COMMAND ${CMAKE_COMMAND} -E tar zxf
-          ${CMAKE_CURRENT_SOURCE_DIR}/master.tar.gz
-          WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
-        file(RENAME "maeparser-${RELEASE_NO}" "${MAEPARSER_DIR}")
-        patchCoordGenMaeExportHeaders("MAEPARSER" "${MAEPARSER_DIR}/MaeParserConfig.hpp")
-
-        # patch a bug in the 1.0.0 release of maeparser
-        #file(READ "${MAEPARSER_DIR}/Buffer.cpp" buffer)
-        #string(REPLACE "m_data->reserve(m_size + 1);"
-        #       "m_data->resize(m_size + 1);" buffer "${buffer}")
-        #file(WRITE "${MAEPARSER_DIR}/Buffer.cpp" "${buffer}")
-    else()
-      message("-- Found MAEParser source in ${MAEPARSER_DIR}")
-    endif()
-
-    file(GLOB MAESOURCES "${MAEPARSER_DIR}/*.cpp")
-    rdkit_library(maeparser ${MAESOURCES} SHARED )
-    install(TARGETS maeparser DESTINATION ${RDKit_LibDir})
 
     if(NOT DEFINED COORDGEN_DIR)
       set(COORDGEN_DIR "${CMAKE_CURRENT_SOURCE_DIR}/coordgenlibs")
     endif()
-    if(NOT EXISTS "${COORDGEN_DIR}/sketcherMinimizer.h")
-        set(RELEASE_NO "1.1")
-        set(SHA "ede3191bf1c1dde53f70ca086014ff9ec7768c55")
-        downloadAndCheckMD5("https://github.com/schrodinger/coordgenlibs/archive/v${RELEASE_NO}.tar.gz"
-              "${CMAKE_CURRENT_SOURCE_DIR}/master.tar.gz"
-              "493cf2eab7d4ab6242e25da112831798")
-        execute_process(COMMAND ${CMAKE_COMMAND} -E tar zxf
-          ${CMAKE_CURRENT_SOURCE_DIR}/master.tar.gz
-          WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
-        file(RENAME "coordgenlibs-${RELEASE_NO}" "${COORDGEN_DIR}")
-        patchCoordGenMaeExportHeaders("COORDGEN" "${COORDGEN_DIR}/CoordgenConfig.hpp")
-    else()
-      message("-- Found coordgenlibs source in ${COORDGEN_DIR}")
-    endif()
-
-    file(GLOB CGSOURCES "${COORDGEN_DIR}/*.cpp")
-    rdkit_library(coordgenlib ${CGSOURCES} SHARED LINK_LIBRARIES maeparser)
-    install(TARGETS coordgenlib DESTINATION ${RDKit_LibDir})
-
-    set(MAE_FILES ${COORDGEN_DIR}/templates.mae )
-    install(FILES ${MAE_FILES}
-            DESTINATION ${RDKit_ShareDir}/Data
-            COMPONENT data )
 
     include_directories(${CMAKE_CURRENT_SOURCE_DIR})
     set(RDK_COORDGEN_LIBS MolAlign coordgenlib maeparser CACHE STRING "the external libraries" FORCE)
--- a/Code/GraphMol/MolInterchange/CMakeLists.txt
+++ b/Code/GraphMol/MolInterchange/CMakeLists.txt
@@ -1,16 +1,5 @@
 
-if(NOT EXISTS "${CMAKE_SOURCE_DIR}/External/rapidjson-1.1.0")
-    downloadAndCheckMD5("https://github.com/Tencent/rapidjson/archive/v1.1.0.tar.gz"
-          "${CMAKE_SOURCE_DIR}/External/rapidjson-1.1.0.tar.gz"
-          "badd12c511e081fec6c89c43a7027bce")
-    execute_process(COMMAND ${CMAKE_COMMAND} -E tar zxf
-      ${CMAKE_SOURCE_DIR}/External/rapidjson-1.1.0.tar.gz
-      WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}/External)
-else()
-  message("-- Found RapidJSON source in ${CMAKE_SOURCE_DIR}/External")
-endif()
-
-include_directories(${CMAKE_SOURCE_DIR}/External/rapidjson-1.1.0/include)
+include_directories(/usr/include/rapidjson)
 
 
 remove_definitions(-DRDKIT_GRAPHMOL_BUILD)
--- a/Code/GraphMol/CMakeLists.txt
+++ b/Code/GraphMol/CMakeLists.txt
@@ -13,7 +13,7 @@
               LINK_LIBRARIES RDGeometryLib RDGeneral
                  ${RDKit_THREAD_LIBS})
 
-target_link_libraries(GraphMol PUBLIC ${RDKit_THREAD_LIBS})
+target_link_libraries(GraphMol PUBLIC coordgenlibs maeparser ${RDKit_THREAD_LIBS})
 
 rdkit_headers(Atom.h
               atomic_data.h
