Index: rdkit/Code/PgSQL/rdkit/adapter.cpp
===================================================================
--- rdkit.orig/Code/PgSQL/rdkit/adapter.cpp
+++ rdkit/Code/PgSQL/rdkit/adapter.cpp
@@ -739,7 +739,7 @@ extern "C" CROMol MolAdjustQueryProperti
     try {
       MolOps::parseAdjustQueryParametersFromJSON(p, pstring);
     } catch (const ValueErrorException &e) {
-      elog(ERROR, e.what());
+      elog(ERROR, "%s", e.what());
     } catch (...) {
       elog(WARNING,
            "adjustQueryProperties: Invalid argument \'params\' ignored");
Index: rdkit/Code/PgSQL/rdkit/mol_op.c
===================================================================
--- rdkit.orig/Code/PgSQL/rdkit/mol_op.c
+++ rdkit/Code/PgSQL/rdkit/mol_op.c
@@ -645,7 +645,7 @@ fmcs_transition(PG_FUNCTION_ARGS) {
 //  int      tsmiles_len = VARSIZE(tsmiles);
   char    *smiles = pgstrdup(VARDATA(tsmiles), VARSIZE(tsmiles));
   char    *params = PG_GETARG_CSTRING(2);
-elog(WARNING, smiles);
+//elog(WARNING, smiles);
 //  Assert(smiles != 0);
     if (AggCheckCallContext(fcinfo, NULL))
     {
