--- a/Code/PgSQL/rdkit/adapter.cpp
+++ b/Code/PgSQL/rdkit/adapter.cpp
@@ -646,7 +646,7 @@
     try {
       MolOps::parseAdjustQueryParametersFromJSON(p, pstring);
     } catch (const ValueErrorException &e) {
-      elog(ERROR, e.message());
+      elog(ERROR, "%s", e.message());
     } catch (...) {
       elog(WARNING,
            "adjustQueryProperties: Invalid argument \'params\' ignored");
--- a/Code/PgSQL/rdkit/mol_op.c
+++ b/Code/PgSQL/rdkit/mol_op.c
@@ -613,7 +613,7 @@
 //  int      tsmiles_len = VARSIZE(tsmiles);
   char    *smiles = pgstrdup(VARDATA(tsmiles), VARSIZE(tsmiles));
   char    *params = PG_GETARG_CSTRING(2);
-elog(WARNING, smiles);
+//elog(WARNING, smiles);
 //  Assert(smiles != 0);
     if (AggCheckCallContext(fcinfo, NULL))
     {
--- a/Code/PgSQL/rdkit/bfp_gist.c
+++ b/Code/PgSQL/rdkit/bfp_gist.c
@@ -33,7 +33,6 @@
 #include <fmgr.h>
 #include <access/gist.h>
 #include <access/skey.h>
-#include <access/tuptoaster.h>
 #include <utils/memutils.h>
 
 #include <math.h>
--- a/Code/PgSQL/rdkit/low_gist.c
+++ b/Code/PgSQL/rdkit/low_gist.c
@@ -33,7 +33,6 @@
 #include <fmgr.h>
 #include <access/gist.h>
 #include <access/skey.h>
-#include <access/tuptoaster.h>
 #include <utils/memutils.h>
 
 #include "rdkit.h"
--- a/Code/PgSQL/rdkit/rdkit_gist.c
+++ b/Code/PgSQL/rdkit/rdkit_gist.c
@@ -32,7 +32,6 @@
 #include <fmgr.h>
 #include <access/gist.h>
 #include <access/skey.h>
-#include <access/tuptoaster.h>
 #include <utils/memutils.h>
 
 #include "rdkit.h"
