--- a/Docs/Book/conf.py
+++ b/Docs/Book/conf.py
@@ -27,7 +27,7 @@
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'recommonmark']  # , 'extapi']
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.doctest', 'sphinx.ext.imgconverter', 'recommonmark']  # , 'extapi']
 #autosummary_generate = True
 doctest_test_doctest_blocks = ""
 
